### PR TITLE
org.supercsv.example.Reading is not working on both eclise and idea

### PR DESCRIPTION
**Pull request by dota17**
Fix ISSUE #136

### CODE REVIEW

**Comment by kbilsted** [comment]
I've installed eclipse and will try to see that I can still execute the tests with the PR. Anyone else have any luck with the PR?

**Comment by dota17** [comment]
@kbilsted  yep it is ok in eclipse, but it is not in IDEA. caused by the different relative path of the IDEA and ECLIPSE projects

**Comment by dota17** [comment]
@kbilsted  and this issue has two problem in fact.  
the first is :
    the relative path of csv file, caused by different of eclipse and IntelliJ IDEA
   private static final String CSV_FILENAME = "src/test/resources/customers.csv";
   need to as following for idea
   private static final String CSV_FILENAME = "super-csv/src/test/resources/customers.csv";
the second is:
    In method getProcessors() need to add
   new ParseSqlTime("HH:mm:ss"), // birthTime  
   the example code is not match with the customers.csv

**Comment by kbilsted** [comment]
@dota17  yes, I just wanted to try it out on my machine first. But got stuck on mvn not playing nice with JDK 11. Hence https://github.com/super-csv/super-csv/pull/169

File: super-csv/src/test/java/org/supercsv/example/Reading.java [hunk 1]
```diff
@@ -70,6 +73,7 @@ public static void main(String[] args) throws Exception {
 			new NotNull(), // firstName
 			new NotNull(), // lastName
 			new ParseDate("dd/MM/yyyy"), // birthDate
+			new ParseSqlTime("HH:mm:ss"),
```
Comment (kbilsted) [inline super-csv/src/test/java/org/supercsv/example/Reading.java]: why are we adding new processers if we are fixing a filepath? The PR is smaller and easier to understand if it only fixes one thing. But maybe there is a good reason..

Comment (dota17) [reply super-csv/src/test/java/org/supercsv/example/Reading.java]: In fact, two problems have been fixed here.
first: filePath problem.
second: 11 columns in file. However 10 CellProcessors in code.
Do I need to change this to two PR to submit?

**Pull request merged**